mithril.js

```diff
@@ -248,8 +248,21 @@ var m = (function app(window, undefined) {
 			else if (data.tag === "svg") namespace = "http://www.w3.org/2000/svg";
 			else if (data.tag === "math") namespace = "http://www.w3.org/1998/Math/MathML";
 			if (isNew) {
-				if (data.attrs.is) node = namespace === undefined ? $document.createElement(data.tag, data.attrs.is) : $document.createElementNS(namespace, data.tag, data.attrs.is);
-				else node = namespace === undefined ? $document.createElement(data.tag) : $document.createElementNS(namespace, data.tag);
+				if (m.redraw.strategy() === "existing") {
```

Comment:
This requires some testing. It is here at all because browsers insert nodes not explicitly included in html, eg tbody. It might make more sense to handle this in the ARRAY branch, ie figure out who the parent of the node actually is and pass that down to the recursive call to build.